cmd/kube-apiserver/app/server.go

```diff
@@ -455,6 +456,7 @@ func (s *APIServer) Run(_ []string) error {
 						glog.Infof("Using self-signed cert (%s, %s)", s.TLSCertFile, s.TLSPrivateKeyFile)
 					}
 				}
+				_ = systemd.SdNotify("READY=1\n")
```

Comment:
Is there any reason to ignore the SdNotify error here? Even if we don't plan to fail, we should log it? 

Another solution might be to check if we are running under systemd, maybe using something like [dbus.New()](http://godoc.org/github.com/coreos/go-systemd/dbus#New) and checking for an error.